core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiListOptionsSpec.scala

```diff
@@ -230,6 +229,8 @@ class WorkspaceApiListOptionsSpec extends ApiServiceSpec {
             Set(
               WorkspaceListResponse(
                 WorkspaceAccessLevels.Owner,
+                Some(true),
```

Comment:
If the user doesn't specify a list of fields to return, the complete list is returned (meaning that you basically have to opt OUT of receiving `canShare` and `canCompute`.